mautic/mautic
# Extend point action with tagging

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | No
| New feature? | Yes
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #4736
| BC breaks? | No
| Deprecations? | No

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Now mautic users can add tags to point actions. These tags are relational data from 'lead_tags' table. When point action triggers for any lead corresponding tags will be tagged.

[//]: # ( As applicable: )
#### Steps to test this PR:
1. Add tags to point action
2. Check lead tags after triggering above point action
